src/test/java/org/cbioportal/session_service/SessionServiceTest.java

```diff
@@ -263,8 +279,10 @@ public void getSessionWithQueryFieldStartsWithDollarSign() throws Exception {
         String data = "\"portal-session\":{\"title\":\"my portal session\"}";
         ResponseEntity<String> response = addData("msk_portal", "main_session", data);
 
+        HttpEntity<String> entity = prepareData("\"$data.portal-session.title\":\"my portal session\"");
+
         // now query
-        response = template.getForEntity(base.toString() + "msk_portal/main_session/" + "query?field=$data.portal-session.title&value=my portal session", String.class);
+        response = template.exchange(base.toString() + "msk_portal/main_session/query/fetch", HttpMethod.POST, entity, String.class);
```

Comment:
@kalletlak maybe good to have separate test for both POST and GET? I assume the old GET is still working?